src/assets/view/options.vue

```diff
@@ -635,6 +635,19 @@
                     </v-slider>
                   </x-list-item>
                   <v-divider></v-divider>
+                  <x-list-item label="Theme">
+                    <v-chip-group
+                      v-model="settings.theme_mode"
+                      @change="onThemeChange"
```

Comment:
Use the `saveSettings` method to save the settings.

```suggestion
                      @change="saveSettings({ theme_mode: $event })"
```